lib/Frontend/CompilerInvocation.cpp

```diff
@@ -364,8 +364,7 @@ static bool ParseLangArgs(LangOptions &Opts, ArgList &Args,
 
   Opts.EnableSwift3ObjCInference =
     Args.hasFlag(OPT_enable_swift3_objc_inference,
-                 OPT_disable_swift3_objc_inference,
-                 Opts.isSwiftVersion3());
+                 OPT_disable_swift3_objc_inference, false);
```

Comment:
I hope we can kill this, too. Is that a separate task?

Reply:
Yeah, I’d like to consider it separate since it’s pretty self contained.